clang-tools-extra/test/clang-doc/basic-project.test

```diff
@@ -384,3 +410,4 @@
 
 // MD-INDEX: #  C/C++ Reference
 // MD-INDEX: * Namespace: [GlobalNamespace](GlobalNamespace)
+ 
```

Comment:
I'm guessing the line ending is CRLF here instead of just a new or something. the red symbol won't show up if the last line is empty and hast the correct line ending.